tools/src/main/scala/scala/scalanative/codegen/Lower.scala

```diff
@@ -1038,6 +1113,10 @@ object Lower {
   val throwSig  = Type.Function(Seq(Type.Ptr), Type.Nothing)
   val throw_    = Val.Global(throwName, Type.Ptr)
 
+  val toClassName = Rt.Runtime.name.member(Rt.ToClassSig)
+  val toClassSig  = Type.Function(Seq(Rt.Runtime, Type.Ptr), Rt.Class)
+  val toClass     = Val.Global(toClassName, Type.Ptr)
```

Comment:
I'm not sure, but I think this should be similar to how we deal with other methods of `runtime`, for example `throwOutOfBounds` down below. (both the definitions of these constants, and how we use then in `buf.call` in `genClassOf` -- although we don't need the `unwind` handler like in the methods that throw)